content/applications/inventory_and_mrp/manufacturing/management/work_order_dependencies.rst

```diff
@@ -0,0 +1,118 @@
+the :guilabel:`Operations` tab.
+
+.. image:: work_order_dependencies/operation-dependencies.png
+   :align: center
+   :alt: The Operation Dependencies checkbox on the Miscellaneous tab of a BoM.
+
+Next, click on the :guilabel:`Operations` tab. On the top right of the tab, click on the tab's
+:guilabel:`settings` button, then enable the :guilabel:`Blocked By` checkbox. This makes a
+:guilabel:`Blocked By` field appear for each operation on the :guilabel:`Operations` tab.
+
+.. image:: work_order_dependencies/operations-settings.png
+   :align: center
+   :alt: The settings for the Operations tab on a BoM.
+
+In the line of the operation that should be blocked by another operation, click the
+:guilabel:`Blocked By` field, and a :guilabel:`Open: Operations` pop-up window appears. In the
+:guilabel:`Blocked By` drop-down field on the pop-up window, select the operation that must be
+completed before the operation that is blocked.
+
+.. image:: work_order_dependencies/blocked-by.png
+   :align: center
+   :alt: The Blocked By drop-down field for an operation on a BoM.
+
+Finally, save the |BOM| by clicking :guilabel:`Save`.
+
+Plan work orders using dependencies
+===================================
+
+Once work order dependencies have been configured on a |BOM|, Odoo *Manufacturing* is able to plan
+when work orders will be scheduled based on their dependencies. To plan the work orders for a
+manufacturing order, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`.
+
+Next, select a manufacturing order for a product with work order dependencies set on its BoM, or
+create a new manufacturing order by clicking :guilabel:`New`. If a new manufacturing order is
+created, select a |BOM| configured with work order dependencies from the :guilabel:`Bill of
+Material` drop-down field, then click :guilabel:`Confirm`.
+
+After confirming the manufacturing order, select the :guilabel:`Work Orders` tab to view the work
+orders required to complete it. Any work orders that are not blocked by a different work order
+display a `Ready` tag in the :guilabel:`Status` section.
+
+Work orders that are blocked by one or more work orders display a `Waiting for another WO` tag
+instead. Once the blocking work order(s) are completed, the tag updates to `Ready`.
+
+.. image:: work_order_dependencies/work-order-status.png
+   :align: center
+   :alt: The status tags for work orders on a manufacturing order.
+
+To schedule the manufacturing order's work orders, click the :guilabel:`Plan` button at the top of
+the page. After doing so, the :guilabel:`Scheduled Start Date` field for each work order on the
+:guilabel:`Work Orders` tab auto-fills with the date and time that the work order is scheduled to
+begin. Work orders that are blocked by a different work order are scheduled at the end of the time
+period specified in the :guilabel:`Expected Duration` field of the work order that precedes it.
```

Comment:
Suggestion for further brevity. Feel free to ignore if this sentence doesn't make as much sense as what you currently have. It's honestly fine as is (:

```suggestion
Blocked work orders
are scheduled after the time period specified in the :guilabel:`Expected Duration` field of the
preceding work order.
```